requirements/requirements.txt

```diff
@@ -7,9 +7,9 @@ python-dateutil
 tensorboardX
 tensorboard
 torch
-torchvision
 wget
 wrapt
 ruamel.yaml
 scikit-learn
 scipy
+hydra-core
```

Comment:
Let's require this lazily - e.g. if user wants to use Hydra only. Kind of like we do with torchaudio. Can we do this?

Reply:
This is clearly a different type of dependency. Let's discuss that tomorrow/today.

Reply:
decided on core req